app/src/main/java/com/keylesspalace/tusky/FiltersActivity.kt

```diff
@@ -0,0 +1,179 @@
+package com.keylesspalace.tusky
+
+import android.app.AlertDialog
```

Comment:
You need to use the import from AndroidX to get the right dialog styling.

```suggestion
import androidx.appcompat.app.AlertDialog
```

Reply:
You can check out how list editing dialog is done (it's very similar dialog):
https://github.com/tuskyapp/Tusky/pull/1104/files#diff-4bf8dc8fd7cbf714a810d8653cdfae55R116
code is not pretty but it works.